src/Redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+const categoriesReducer = (state = [], action) => {
+  if (action.type === CHECK_STATUS) {
+    return ['Under Construction'];
```

Comment:
- [ ] In keeping with the requirement this should be a string, not an array. Kindly refactor accordingly.